main.py

```diff
@@ -307,6 +319,13 @@ def hotkey(self, event):
         elif event.char == "z":
             print("Deleting last box...");
             self.frame_canvas.clear_last_label()
+        elif event.char == "r":
+            self.frame_canvas.rollover_label = True
+            print(f"r pressed rollover is now {self.frame_canvas.rollover_label}")
+        elif event.char == "t":
```

Comment:
Maybe add debug text here too, but I still second using just 'r' as toggle for simplicity

Reply:
Also, make the print statement non-format string to be consistent with rest of code.